DisguiseUnityRenderStream/Runtime/PluginEntry.cs

```diff
@@ -403,50 +403,20 @@ public RS_ERROR getStreams(ref StreamDescription[] streams)
             return res;
         }
 
-        public RS_ERROR sendFrame(UInt64 streamHandle, SenderFrameType frameType, SenderFrameTypeData data, FrameResponseData sendData)
+        public RS_ERROR sendFrame(UInt64 streamHandle, SenderFrameType frameType, SenderFrameTypeData data, ref FrameResponseData sendData)
```

Comment:
Personal education: Why are we passing sendData by ref instead of the normal "by copy"?  Is it to avoid the cost of the copy or there is another reason (since from what I understand m_sendFrame should not modify it)?

Reply:
Yeah, it's to avoid a copy. It sucks that C# doesn't have const refs and `in` parameters don't work the way you think they do 😞